skymp5-front/src/interfaces/buttons.ts

```diff
@@ -5,6 +5,14 @@ export interface SkyrimHintProps {
     left: boolean;
 }
 
+export interface SkyrimInputProps extends DefaultButtonComponentProps {
+    labelText: string;
+    type: React.HTMLInputTypeAttribute;
+    initialValue: string | number | readonly string[];
+    placeholder: string;
+    onInput?: (any?: any) => void;
```

Comment:
there can be concrete type
`onInput?: (text: string) => void | () => void | (isChecked: boolean) => void`

Reply:
So there is still `any`